database/factories/ModelFactory.php

```diff
@@ -53,10 +53,12 @@
     ];
 });
 
-$factory->define(App\DrugReview::class, function (Faker\Generator $faker) {
+$max_drug_id = \Illuminate\Support\Facades\DB::table('drugs')->max('id');
```

Comment:
This line forces a DB query whenever this factory is instantiated, so it breaks some tests that did not need a real db connection so far.